style.css

```diff
@@ -0,0 +1,94 @@
+.site-header {
+    padding: 10px;
+    border-top: 1px solid black;
+    border-bottom: 8px double aquamarine;
+    margin-bottom: 8px;
+    margin-left: 10px;
+    margin-right: 10px;
+}
+.shopping-card {
+    box-sizing: border-box;
+    
+    border-radius: 7px;
+    overflow: hidden;
+    display: flex;
+    flex-direction: row;
```

Comment:
to jest domyslna wartosc, nie trzeba jej wpisywac, jak piszesz jakis kod to potem warto sobie odpalic inspektor kodu w przegladarce i probowac wlaczac / wylaczac poszczegolne reguly i sprawdzac czy sa one potrzebne, czesto mozna cos przypadkowo nadmiarowo napisac i zostawic